### ISSUE TITLE
No permission that maps to product_media:update

### ISSUE THREAD

**Issue by nickygerritsen** [shopware/platform#2905]
### PHP Version

8.1

### Shopware Version

6.4.18

### Expected behaviour

When I assign the 'Edit' permission on product (or media?) I should be able to change media of a product in the API.

### Actual behaviour

I get the error 

```json
{"message":"Missing privilege","missingPrivileges":["product_media:update","product_media:update","product_media:update"]}
````

### How to reproduce

Create a role with the above permissions and try to patch product media.

I think the `product_media:read` permission should be added to one of these admin permissions.

**Comment by Zwaen91** [comment]
@nickygerritsen I reproduced your problem and had no problems with the permissions "product:update" and "media:read". If you want to assign a media to a product the "product:update" permission is the required one. It's misleading that a error message regarding the media is exposed.

**Comment by nickygerritsen** [comment]
Let me try again to verify what went wrong and come back. Maybe it was a PEBKAC indeed

**Comment by nickygerritsen** [comment]
Seems to work with 6.5 RC1 indeed, not sure what changed. Note that I also found the "Detailed privileges" tab that could help me out for specific permissions.
